GVFS/GVFS.Common/Git/GitProcess.cs

```diff
@@ -83,6 +83,8 @@ public GitProcess(string gitBinPath, string workingDirectoryRoot, string gvfsHoo
             }
         }
 
+        public bool ForBackground { get; set; }

```

Comment:
I would consider including a comment about what this means. Background can be a general without context... e.g. that this process will be run with lower priority to not impact other uses of the machine..

Reply:
I'll just rename it to `LowerPriority` so the name is self-documenting.